src/app/component/layout/footer/footer.hbs

```diff
@@ -1,5 +1,5 @@
 <link rel="stylesheet" href="./footer.scss">
```

Comment:
If you've decided to do XML style closing tags on this ` <link rel="stylesheet" href="./footer.scss" />`